Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:latest
+MAINTAINER nelluri@deis.com
+ENV CLOUD_SDK_REPO cloud-sdk-precise
+RUN echo "deb http://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" | tee /etc/apt/sources.list.d/google-cloud-sdk.list
+RUN curl -L https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+RUN apt-get update && apt-get -y install make python lsb-release unzip wget vim git curl jq google-cloud-sdk
+RUN wget https://storage.googleapis.com/kubernetes-release/release/v1.0.3/bin/linux/amd64/kubectl
+RUN chmod +x kubectl && mv kubectl /usr/local/bin
+WORKDIR /home
+RUN wget https://dl.bintray.com/deis/helm/helm-0.5.0+1689ee4-linux-amd64.zip
+RUN unzip helm-0.5.0+1689ee4-linux-amd64.zip
+RUN mv helm /usr/local/bin/
+RUN mkdir /home/charts
+RUN mkdir /root/.secrets
+WORKDIR /home/charts
+ENV TERM xterm-256color
+ADD . .
+RUN curl -L https://dl.bintray.com/sgoings/rerun-modules/rerun-0.2.0-10-g7340992 > rerun
+RUN chmod +x rerun 
+RUN echo "alias init="/home/charts/_scripts/tests/init.sh"" >> $HOME/.bashrc
```

Comment:
Wouldn't it be easier just to add `/home/charts/_scripts/tests` to $PATH instead?